src/test/java/com/microtripit/mandrillapp/lutung/controller/MandrillMessagesApiTest.java

```diff
@@ -31,7 +26,7 @@
 import com.microtripit.mandrillapp.lutung.view.MandrillSearchMessageParams;
 import com.microtripit.mandrillapp.lutung.view.MandrillTemplate;
 
-import junit.framework.Assert;
```

Comment:
repaced `junit.framework.Assert` (deprecated) with the non-deprecated `org.junit.Assert`